src/sentry/static/sentry/app/views/organizationIntegrations/pluginDetailedView.tsx

```diff
@@ -305,6 +313,18 @@ const DisabledNotice = styled(({reason, ...p}: {reason: React.ReactNode}) => (
   font-size: 0.9em;
 `;
 
+const StatusWrapper = styled('div')`
```

Comment:
@NisanthanNanthakumar It seems that we have three identical definitions of `StatusWrapper`. Wondering if there is a way so we don't have to repeat this in multiple files. Would it make sense to make a new component for this? Or pass in a prop to `IntegrationStatus` that would apply this styling?

Reply:
@scefali when we refactor the detailed views we can combine it into one StatusWrapper. But this styled component should stay in the detailed view page.

Reply:
Ok, sounds good to me